schemainspect/pg/obj.py

```diff
@@ -943,6 +944,49 @@ def key(self):
         return self.object_type, self.quoted_full_name, self.target_user, self.privilege
 
 
+class InspectedComment(Inspected):
+    def __init__(self, object_type, schema, table, name, args, comment):
+        self.object_type = object_type
+        self.schema = schema
+        self.table = table
+        self.name = name
+        self.args = args
+        self.comment = comment
+
+    @property
+    def _identifier(self):
+        return quoted_identifier(
+            self.name,
+            schema=self.schema,
+            table=self.table,
+            identity_arguments=self.args,
+        )
+
+    @property
+    def drop_statement(self):
+        return "comment on {} {} is null;".format(self.object_type, self._identifier)
+
+    @property
+    def create_statement(self):
+        return "comment on {} {} is '{}';".format(
```

Comment:
```suggestion
        return "comment on {} {} is $cmt${}$cmt$;".format(
```
Might be worth using dollar quoting to allow unescaped `'` in the comment